test/BatchManager-test.js

```diff
@@ -2,7 +2,7 @@ import { assert } from 'chai';
 import sinon from 'sinon-sandbox';
 
 import { makeJob, COMPONENT_NAME } from './helper';
-import BatchManager from '../lib/utils/BatchManager';
+import BatchManager from '../src/utils/BatchManager';
```

Comment:
Similarly, we actually *want* tests to be run on babel output; not on the raw source.